ddtrace/constants.py

```diff
@@ -22,6 +22,15 @@
 
 LOG_SPAN_KEY = "__datadog_log_span"
 
+ERROR_MSG = "error.msg"  # a string representing the error message
+ERROR_TYPE = "error.type"  # a string representing the type of the error
+ERROR_STACK = "error.stack"  # a human readable version of the stack. beta.
```

Comment:
```suggestion
ERROR_STACK = "error.stack"  # a human readable version of the stack
```

hasn't been "beta".... for like 4 years 😆

Reply:
👍  will fix this with the other suggestions